lib/SparkPost/SparkPost.php

```diff
@@ -26,7 +26,8 @@ class SparkPost {
       'port'=>443,
       'strictSSL'=>true,
       'key'=>'',
-      'version'=>'v1'
+      'version'=>'v1',
+      'subaccount_id' => 0
```

Comment:
I wouldn't default this to 0.  There are some cases where not setting a subaccount is different than setting it to the master account.